backend/lcfs/web/api/compliance_report/services.py

```diff
@@ -96,6 +99,72 @@ async def create_compliance_report(
 
         return ComplianceReportBaseSchema.model_validate(report)
 
+    @service_handler
+    async def created_analyst_adjustment_report(
+        self, existing_report_id: int, user: UserProfile
+    ) -> ComplianceReportBaseSchema:
+        """
+        Creates a new supplemental compliance report.
```

Comment:
pls update comment to explain what an analyst adjustment supplemental is all about.

Reply:
Update the comments